serverless-inference/serverless-model-registry.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    model_package_group_name = "xgboost-abalone" + strftime("%Y-%m-%d-%H-%M-%S", gmtime())
</pre><p>Style nit - add a blank line between the <code>time</code> imports and following code for readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>